### PR TITLE
Fixes minor errors/updates in Core docs v3.0.2 - Closes #1334

### LINKED ISSUE #1334: Fixes minor errors/updates in Core docs v3.0.2
What:

Fixes minor errors and adds minor updates throughout the Core docs v3.0.2

Description:

Minor spelling and grammatical errors were found throughout the Core docs v3.0.2

Link:
https://lisk.com/documentation/lisk-core/index.html

### PR DESCRIPTION
**Pull request by chris529**
Closes #1334

### CODE REVIEW

File: modules/ROOT/pages/management/account-management.adoc [hunk 1]
```diff
@@ -37,14 +37,14 @@ $ lisk-core account:create
 Return randomly-generated mnemonic passphrase with its corresponding public/private key pair and Lisk address.
 
 USAGE
-  $ lisk-core account:create
+  $ lisk-core account:create.
```
Comment (Tschakki) [inline modules/ROOT/pages/management/account-management.adoc]: These dots don't belong here, it is a command example.

Please remove. Same in the command examples below

File: modules/ROOT/pages/management/enabling-misbehavior-report.adoc [hunk 2]
```diff
@@ -52,7 +52,7 @@ TIP: See more available configuration options at the xref:{url_plugin_report_mis
 
 == Restart the node with the enabled plugin
 
-Start Lisk Core with the flag `--enable-report-misbehavior-plugin `:
+Start Lisk Core with the flag `--enable-report-misbehavior-plugin`
```
Comment (Tschakki) [inline modules/ROOT/pages/management/enabling-misbehavior-report.adoc]: Why remove the colon?

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looking good, thanks! Just two small remarks :)

File: modules/ROOT/pages/management/account-management.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/management/account-management.adoc]: Good point...I have now removed them, as well as other instances in the command examples on this page, so they are now back to their original state 👍

File: modules/ROOT/pages/management/enabling-misbehavior-report.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/management/enabling-misbehavior-report.adoc]: Thanks, that was an error...It's now corrected 👍

File: modules/ROOT/pages/management/account-management.adoc [hunk 3]
```diff
@@ -34,16 +34,16 @@ $ lisk-core account:create
...
 USAGE
   $ lisk-core account:create
 
 OPTIONS
-  -c, --count=count  [default: 1] Number of accounts to create.
+  -c, --count=count  [default: 1] Number of accounts to create
 
 EXAMPLES
-  account:create
+  account:create.
```
Comment (Tschakki) [inline modules/ROOT/pages/management/account-management.adoc]:
```suggestion
  account:create
```

File: modules/ROOT/pages/management/account-management.adoc [hunk 4]
```diff
@@ -172,7 +172,7 @@ OPTIONS
   --pretty                   Prints JSON in pretty format rather than condensed.
 
 EXAMPLE
-  transaction:send
+  transaction:send.
```
Comment (Tschakki) [inline modules/ROOT/pages/management/account-management.adoc]:
```suggestion
  transaction:send
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/management/forging.adoc [hunk 5]
```diff
@@ -89,7 +89,7 @@ The forger_info data contains the following three properties:
 
 For each of these properties, the forger config contains the value used in the last block forged by the node.
 
-The  three variables are required for enabling forging for the corresponding delegate.
+These  three variables are required for enabling forging for the corresponding delegate.
```
Comment (ishantiw) [inline modules/ROOT/pages/management/forging.adoc:92]:
```suggestion
These three variables are required for enabling forging for the corresponding delegate.
```

**Review by ishantiw — APPROVED** [review]
Just a minor comment, LGTM

**Review by Tschakki — APPROVED** [review]
LGTM!

**Pull request merged**